spec/factories.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :log do
+    job_id { 1 }
+    content { 'Hello World!' }
```

Comment:
@speedywizard @AndriiMysko pls put some related content

Reply:
Hello @vitalied 
Could you please provide an example of "related content"?